install-manifests/helm/space-cloud/values.yaml

```diff
@@ -0,0 +1,87 @@
+# Default values for space-cloud.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+clusterId: "default"
+devMode: "true" # Turn this to false for production mode
+version: "0.20.0" # This ensures both runner & gateway using the same version
+
+admin:
+  username: "admin" # Log in username
+  password: "1234" # Log in password
+  secret: "some-secret" # Space cloud uses this secret for parsing jwt tokens for config APIs
+
+meta:
+  logLevel: "debug" # other values info, warn
+  logFormat: "json" # other values text
+
+# Gateway service configuration
+gateway:
+  image:
+    name: "spaceuptech/gateway"
+    pullPolicy: "IfNotPresent" # IfNotPresent | Always
+  resources: 
+    memory: "256Mi"
+    cpu: "200m"
+  ssl:
+    enabled: "true"
+    custom_ssl_secret: "" # If required change this to the Kubernetes secret name containing custom SSL cert
+    custom_ssl_cert_key: "" # Secret key containing SSL public certificate
+    custom_ssl_private_key: "" # Secret key containing SSL private key
+  autoScaler: 
+    averageCPUUtilization: 70
+    minReplicas: 1
+    maxReplicas: 10
+  disableUI: "false"
+  replicas: 3
+  restricted_hosts: "*" # Comma seperated IPs for restricting access to admin UI
+  envs: 
+
+# Runner service configuration
+runner:
+  image:
+    name: "spaceuptech/runner"
+    pullPolicy: "IfNotPresent" # IfNotPresent | Always
+  resources: 
+    memory: "256Mi"
+    cpu: "200m"
+  autoScaler: 
+    averageCPUUtilization: 70
+    minReplicas: 1
+    maxReplicas: 10  
+  envs: 
+
+# Redis service configuration
+redis:
+  image:
+    name: "redis"
+    version: "6.0"
+    pullPolicy: "IfNotPresent" # IfNotPresent | Always
+  resources: 
```

Comment:
I suggest you take a `requests` and `limits` everywhere

Reply:
Should I make it nested?
inside resource object?

Reply:
& what about runner & gateway should I limit there also?
If yes what is the default values